src/components/header.js

```diff
@@ -26,7 +26,7 @@ class Header extends React.Component {
       <header>
         <div className={styles.logoArea}>
           <h3 style={{ margin: 0 }}>
-            <Link to="/">ENGINEERING REVUE</Link>
+            <Link to="/">AUCKLAND ENGINEERING REVUE</Link>
```

Comment:
This should stay as "ENGINEERING REVUE", otherwise it's too long and doesn't fit on mobile.

![image](https://user-images.githubusercontent.com/1442037/79942280-982b9d00-84ba-11ea-8d23-1678b504c818.png)

```suggestion
            <Link to="/">ENGINEERING REVUE</Link>
```